drivers/spi/Kconfig

```diff
@@ -26,21 +26,77 @@ menuconfig SPI
 if SPI
 
 config SPI_SLAVE
-	bool "SPI slave"
+	bool "SPI Slave"
 	default n
 	---help---
-		Enable support for SPI slave features
+		Enable support for SPI Slave features
 
 if SPI_SLAVE
 
+config SPI_SLAVE_DRIVER
+	bool "SPI Slave character driver"
+	default n
+	---help---
+		Built-in support for a character driver at /dev/spislv[N] that may be
+		used to perform SPI bus transfers from applications.
+		The intent of this driver is to support SPI Slave testing.
+
+if SPI_SLAVE_DRIVER
+
+config SPI_SLAVE_DRIVER_MODE
+	int "SPI Slave character driver default mode"
+	default 0
+	---help---
+		Default SPI Slave character driver mode, where:
+		0 = CPOL=0, CPHA=0
+		1 = CPOL=0, CPHA=1
+		2 = CPOL=1, CPHA=0
+		3 = CPOL=1, CPHA=1
+
+config SPI_SLAVE_DRIVER_WIDTH
+	int "SPI Slave character driver default bit width"
+	default 8
+	---help---
+		Number of bits per SPI Slave transfer (default 8).
+
+config SPI_SLAVE_DRIVER_BUFFER_SIZE
+	int "SPI Slave character driver TX and RX buffer sizes"
+	default 128
+	---help---
+		Size of the internal TX and RX buffers of the SPI Slave
+		character driver.
+
+config SPI_SLAVE_DRIVER_COLORIZE_TX_BUFFER
+	bool "SPI Slave character driver colorize TX buffer"
+	default n
+	---help---
+		Initialize entries of the TX buffer with a given pattern.
+		This may be used as the SPI Slave controller first TX transfer.
```

Comment:
```suggestion
		This may be used in the first TX transfer of the SPI Slave controller.
```

Reply:
Thanks, I've applied your suggestion.

Reply:
It may be the intent is that the data can be used?

This data may be used as the SPI Slave controller's first TX transfer.

Reply:
The SPI Slave controller may receive this data through a call to `getdata` function right after performing the `bind` operation.
This way, it could be used as the first TX transfer.

I'll add this info to the help text to make it clearer.

Reply:
@davids5 @saramonteiro PTAL, I've added a more detailed explanation.